run/mc_features.py

```diff
@@ -225,6 +225,8 @@
 ################################################################################
 # Path and EndPath definitions, TFileService, OutputModule
 
+process.dumper = cms.EDAnalyzer('EventContentDumper')
```

Comment:
@mverzett - I see this exception when running:

```
----- Begin Fatal Exception 18-Oct-2018 11:23:28 BST-----------------------
An exception of category 'PluginNotFound' occurred while
   [0] Constructing the EventProcessor
Exception Message:
Unable to find plugin 'EventContentDumper' in category 'CMS EDM Framework Module'. Please check spelling of name.
----- End Fatal Exception -------------------------------------------------
```

Reply:
Oh, crap, this was a quick check I ran, not meant to be committed.